optaplanner-examples/src/test/java/org/optaplanner/examples/pas/solver/BedDesignationDO.java

```diff
@@ -0,0 +1,71 @@
+package org.optaplanner.examples.pas.solver;
+
+import org.optaplanner.examples.pas.domain.AdmissionPart;
+import org.optaplanner.examples.pas.domain.Bed;
+import org.optaplanner.examples.pas.domain.BedDesignation;
+import org.optaplanner.examples.pas.domain.Night;
+import org.optaplanner.examples.pas.domain.Patient;
+import org.optaplanner.examples.pas.domain.Room;
+import org.optaplanner.examples.pas.domain.Specialism;
+
+public class BedDesignationDO extends BedDesignation {
```

Comment:
Personally, I see this class as unnecessary complexity - I'd simply change `BedDesignation` to add the constructors etc.
The other examples do it the same way, so I'd appreciate if we stayed consistent.

Reply:
Ok, just a proposal. Will add with methods

Reply:
https://github.com/kiegroup/optaplanner/pull/872/commits/7dd155181c6f1f7f867667d82eb53de3aa1d5a38